### ISSUE TITLE
AlreadyDisposedException: Cannot create Project, KubernetesConsoleService.getOrCreateKuberenetesExplorerToolWindow

### ISSUE THREAD

**Issue by etanshaul** [GoogleCloudPlatform/cloud-code-intellij#2784]
IDEA 20.1.2
CC 20.6.1

```
Stack trace:
com.intellij.serviceContainer.AlreadyDisposedException: Cannot create Project (name=agency, containerState=DISPOSE_COMPLETED, componentStore=/**/**)  because container is already disposed (container=Project (name=agency, containerState=DISPOSE_COMPLETED, componentStore=/**/**) )
	at com.intellij.serviceContainer.ComponentManagerImpl.checkThatNotDisposedCompletely(ComponentManagerImpl.kt:439)
	at com.intellij.serviceContainer.ComponentManagerImpl.doGetService(ComponentManagerImpl.kt:415)
	at com.intellij.serviceContainer.ComponentManagerImpl.getService(ComponentManagerImpl.kt:394)
	at com.intellij.openapi.wm.ToolWindowManager$Companion.getInstance(ToolWindowManager.kt:19)
	at com.google.cloud.tools.intellij.kubernetes.explorer.KubernetesConsoleService.getOrCreateKuberenetesExplorerToolWindow(KubernetesConsoleService.kt:140)
	at com.google.cloud.tools.intellij.kubernetes.explorer.KubernetesConsoleService.displayProcessStream(KubernetesConsoleService.kt:89)
	at com.google.cloud.tools.intellij.kubernetes.explorer.action.StreamLogsAction$treeActionPerformed$1.invokeSuspend(StreamLogsAction.kt:93)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
```

**Comment by etanshaul** [comment]
Kamino cloned this issue to [GoogleCloudPlatform/cloud-code-intellij-internal](https://github.com/GoogleCloudPlatform/cloud-code-intellij-internal/issues/2085)

**Comment by etanshaul** [comment]
got another report of this.

**Comment by peterlin741** [comment]
Fixed in `20.8.1`
